extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -52,37 +52,14 @@ export class ProjectsController {
 		this.projectTreeViewProvider.load(this.projects);
 	}
 
-	public async openProject(projectFile: vscode.Uri, focusProject: boolean = true, isReferencedProject: boolean = false): Promise<Project> {
-		for (const proj of this.projects) {
-			if (proj.projectFilePath === projectFile.fsPath) {
-				if (!isReferencedProject) {
-					vscode.window.showInformationMessage(constants.projectAlreadyOpened(projectFile.fsPath));
-					return proj;
-				} else {
-					throw new Error(constants.projectAlreadyOpened(projectFile.fsPath));
-				}
-			}
-		}
-
+	public async openProject(projectFile: vscode.Uri): Promise<Project> {
```

Comment:
this openProject() in projectController also should be removed since it isn't being used in the new projects view, but I'll do that separately because so many tests are using it